pegasus/sites.v3/code.org/public/educate/professional-learning.md

```diff
@@ -6,39 +6,58 @@ theme: responsive
+{img: '/images/testimonials/fit-150/teachers-1.jpg', text: 'The majority of our workshop attendees rank it the best professional development ever.'},
+{img: '/images/testimonials/fit-150/teacher-renee.jpg', text: '"I do not have a computer science background. I would change nothing about the training. It was an incredible experience, and I felt valued and respected as a teacher and facilitator. I would absolutely recommend anything Code.org has to offer to any teacher. Period. The lesson plans are incredible."'},
+{img: '/images/testimonials/fit-150/teacher-juan.jpg', text: '"It was absolutely rich and the most meaningful training I have ever attended in 16 years of teaching."'}] %>
+
+### Nationwide workshops for elementary school educators, free of charge
+
+Code.org offers 1-day workshops to prepare educators and content-area teachers (librarians, tech-ed specialists, etc.) to introduce computer science basics in a format that's fun and accessible to the youngest learners (grades K-5). The workshops will cover content for [our elementary school curriculum](/educate/curriculum/elementary-school), and you will receive the supplies you need to teach the courses - at no cost. You don't need any previous experience teaching computer science to join us. Not only is this an opportunity to increase your knowledge and confidence in facilitating an engaged computer science classroom, it's also a chance to connect with other educators in your area who are working to bring computer science to their schools.
+
+These workshops are facilitated by experienced computer science educators across the United States.
 
 [<button>Learn more</button>](/k5) &nbsp;&nbsp;[<button>Find a local workshop (for K-5 teachers)</button>](/professional-development-workshops)
+
 <br><br>
-Can't find a workshop in your area? Consider completing our free, self-paced [online workshop.](/educate/professional-development-online)
 
----
-## Professional learning for middle and high school teachers to teach computer science
+### Professional learning for middle and high school teachers to teach computer science
 
-Our middle and high school programs help school districts offer full-year computer science classes by preparing existing teaching staff to offer the course. This multi-phased professional learning program is available in areas where Code.org has a [Regional Partner](/educate/partner).
+Our middle and high school programs offer year-round support. The program kicks off with a 5-day summer workshop where you'll have an opportunity to work hands-on with the curriculum and meet other teachers from your area. Throughout the year, we offer online support for upcoming units, forum support, and 1-day quarterly workshops. You don't need any prior computer science experience to get started.
 
+<br><br>
 
-### Summer Workshop
-Summer 5 day in-person professional learning workshop, followed by online material.
+#### 7-9th grade teachers: Computer Science Discoveries
```

Comment:
nit: The big gaps and our heading styles made me not quite grok whether these two sections were inside the middle/high school section.